stl/inc/coroutine

```diff
@@ -14,7 +14,7 @@
     "The contents of <coroutine> are not available with /await. "  \
     "Remove /await or use /await:strict for standard coroutines. " \
     "Use <experimental/coroutine> for legacy /await support."
-#pragma message(_WARNING_MESSAGE("STL4039", _STL4039_COROUTINE))
+_EMIT_STL_WARNING("STL4039", _STL4039_COROUTINE));
 #undef _STL4039_COROUTINE // TRANSITION, DevCom-1479701
 #else // ^^^ /await ^^^ / vvv no /await vvv
 #ifndef __cpp_lib_coroutine

```

Comment:
The tests `Dev09_119644_compiler_option_gz`, `GH_001411_core_headers`, and `GH_002711_Zc_alignedNew-` are failing due to line 22:

```
D:\build\out\inc\coroutine(22): error C2220: the following warning is treated as an error
D:\build\out\inc\coroutine(22): warning C4083: expected 'string'; found identifier '_WARNING_MESSAGE'
```

Reply:
ah, that wasn't found by my regex, thanks!